app/src/renderer/node.js

```diff
@@ -61,6 +62,28 @@ module.exports = function (nodeIP) {
 
       node.rpcConnecting = false
       return nodeIP
+    },
+    async sendTx (args, account, password) {
```

Comment:
why is this in `node.js`?

Reply:
I feel like this might not be the right module, although I understand why `wallet` might not have felt right either. Maybe this could just be its own `tx` module, with only this action and the queue of transactions plus the sequence number?